### PR TITLE
Centralize metric and logging docs

### PR DESCRIPTION
**Pull request by ericmj**

### CODE REVIEW

File: lib/phoenix/logger.ex [hunk 1]
```diff
@@ -1,6 +1,58 @@
...
+
+    * `[:phoenix, :error_rendered]` - dispatched at the end of an error view being rendered
+      * Measurement: `%{duration: native_time}`
+      * Metadata: `%{status: Plug.Conn.status, kind: Exception.kind, reason: term, stacktrace: Exception.stacktrace}`
+      * Disable logging: Set `render_errors: [log: false]` on your endpoint configuration
+
+    * `[:phoenix, :socket_connected]` - dispatched by `Phoenix.Socket`, at the end of a socket connection
+      * Measurement: `%{duration: native_time}`
+      * Metadata: `%{endpoint: atom, transport: atom, params: term, connect_info: map, vsn: binary, user_socket: atom, result: :ok | :error, serializer: atom, log: Logger.level | false}`
+      * Disable logging: `use Phoenix.Socket, log: false`
```
Comment (ericmj) [inline lib/phoenix/logger.ex]: This should be configurable for liveview sockets but is currently not possible.

File: lib/phoenix/logger.ex [hunk 2]
```diff
@@ -1,6 +1,58 @@
...
+
+    * `[:phoenix, :channel_joined]` - dispatched at the end of a channel join
+      * Measurement: `%{duration: native_time}`
+      * Metadata: `%{params: term, socket: Phoenix.Socket.t}`
+      * Disable logging: This event cannot be disabled
```
Comment (ericmj) [inline lib/phoenix/logger.ex]: Should we able to disable logging of channel events?

Comment (josevalim) [reply lib/phoenix/logger.ex]: If someone requests it, sure!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**